ui/appui-react/src/appui-react/messages/MessageManager.tsx

```diff
@@ -221,6 +228,51 @@ export class MessageManager {
     MessageManager.addMessage(message);
   }
 
+  public static registerAnimateOutRef(el: HTMLElement | null) {
+    this.animateOutToRef = el;
+  }
+
+  /** Display a message.
+   * Works only with `Sticky` and `Toast` message types.
+   * @param  message  Details about the message to display.
+   * @param options Optionally override individual toast parameters.
+   * @param settings Optionally override all toasts settings (i.e. placement or order).
+   * @returns Object with reference to the message (i.e. to close it programmatically) if it was displayed.
+   */
+  public static displayMessage(message: NotifyMessageDetailsType, options?: ToastOptions, settings?: ToasterSettings) {
```

Comment:
Instead of rendering Messages in different components like before, have a single method to call to get the toasts.
This is applicable only to `Sticky` and `Toast` types.